### ISSUE TITLE
Issues after updating to lutris-0.5.8.3-1.fc33.x86_64

### ISSUE THREAD

**Issue by rijan72** [lutris/lutris#3397]
**Issue with cursor**

I had been playing Yakuza 0 from lutris.
Everything was working fine and I am much thankful for this software.

The issue was not there till the latest update. At first I had though it was a bug inside the game so I had uninstall and reinstall the game multiple times (check the screenshot) but the issue remained at every point. 

The issue is hard to describe as the mouse works as it should in context menus and Esc page. The issue come when playing the character where left swipe becomes right/ right swipe become left / up swipe become ?? / down swipe become ??.  At first I thought this was a game bug as the mouse worked fine in the menus so I would relaunch the game, reinstall the game, restart the laptop, unplug replug mouse.

In the end what worked after much frustration was simply downgrading lutris-0.5.8.3-1.fc33.x86_64 to  lutris-0.5.7.1-3.fc33.x86_64

I am unclear about the cause but that was the fix.

**Steps to reproduce**

-  update lutris-0.5.7.1-3.fc33.x86_64 to lutris-0.5.8.3-1.fc33.x86_64 
-  start the game with same config 
-  launch saved or new game 
-  looking around . 

**Lutris debugging output (Optional)**

The terminal output of `lutris -d`.

Unable to load locale dir, translations won't work.
2021-01-30 13:57:00,333: Getting full game list from MAME...
2021-01-30 13:57:00,333: MAME XML generation launched in the background, not returning anything this time
2021-01-30 13:57:00,333: MAME isn't installed, can't retrieve systems list.
2021-01-30 13:57:01,340: MAME XML written

Using the debug option reveals possible backtraces or missing libraries. This can help us track down your issue.

**System information (Optional)**
![Screenshot from 2021-01-30 14-00-53](https://user-images.githubusercontent.com/34618572/106351319-a3421200-6303-11eb-8780-04a1160385c2.png)

**Screenshots (Optional)**
![Screenshot from 2021-01-30 13-51-39](https://user-images.githubusercontent.com/34618572/106351258-4ba3a680-6303-11eb-8417-949cdffab9b2.png)
![Screenshot from 2021-01-30 13-52-21](https://user-images.githubusercontent.com/34618572/106351259-4d6d6a00-6303-11eb-87e7-fe142985975f.png)
![Screenshot from 2021-01-30 13-59-26](https://user-images.githubusercontent.com/34618572/106351272-65dd8480-6303-11eb-9c4c-743ebb8bbc4f.png)

If your issue is hard to explain with words, feel free to attach screenshots.

**Comment by rijan72** [comment]
As this is my first this opening a issue I may have left out or over written something. I would be glad to submit any required information. Thank you.

**Comment by strycore** [comment]
There is no information in this report I can use to help you. Sorry.

**Comment by rijan72** [comment]
I guess I'll have to update lutris only  after completing the game.

**Comment by rijan72** [comment]
This issue has somehow been solved.
Previously I had updated system/lutris using [ dnf update ] which does not require system restart.
My plan was to exclude lutris from updating until I had completed the gameplay of Yakuza 0 using dnf.conf but I had forgotten to modify the dnf.conf file.
So when I had shutdown the pc in the next boot my system  [ gnome-software restart to update thing ] had auto-updated lutris.

This restart to update has somehow removed the above mentioned issue.
